ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -300,6 +300,39 @@ echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 openshift-install --dir="${dir}" create manifests &
 wait "$!"
 
+# Adds arm64 workers for heterogeneous deployment 
+# TO-DO Remove once we have IPI support for heterogeneous in 4.13 https://issues.redhat.com/browse/MIXEDARCH-143 
+if [ ! -z "$ADDITIONAL_WORKERS" ]; then
```

Comment:
This needs to be a new step. See https://github.com/openshift/release/blob/48c1af152e2ddee2e0b698212ef94ff337030fe8/ci-operator/step-registry/ingress/azure/custom-ingresscontroller/ingress-azure-custom-ingresscontroller-commands.sh#L44-L105 as an example of custom MachineSet and https://github.com/openshift/release/blob/826a02f24699747244b64524f93a9861f7e4ba60/ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh for an example of mixing in additional manifest for installer

Reply:
Updated